src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -39,7 +43,7 @@ const useSaveAnonChatHook = () => {
     type: string
   ) => {
     if (!localDb) return;
-    const {channelName, channelImage, originalMembers} = await getChannelListInfo(
+    const {channelName, channelImage, originalMembers} = getChannelListInfo(
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `await` before `getChannelListInfo` might cause issues if this function is asynchronous. If it returns a Promise, you should await it to ensure that the data is available when you need it.



<!-- This is an auto-generated comment by OSS CodeRabbit -->